app/views/episodes/edit.slim

```diff
@@ -0,0 +1,73 @@
+.container
+  .py-3.text-center
+    img.d-block.mx-auto.mb-4.logo[src="/proconf_logo.jpg" alt="proConf logo"]
+    h2
+      | Let's update podcast
+    p.lead
+      | Below fill form fields to ...
+.col-md-12.order-md-1
+  h4.mb-3
+    | Announcement data
+  form.needs-validation[method='post' action="/episodes/#{@episode.id}" novalidate]
+    .row
+      .col-md-4.mb-3
+        label[for="lastName"]
+          | Start datetime
+        input.form-control[name='date' type="date" value="#{@episode.date.time.to_s.split(' ').first}" required]
```

Comment:
Label for lastname) move this logic to model or better create new field
@episode.date.time.to_s.split(' ').first/second